README.md

```diff
@@ -4,13 +4,30 @@
 [![Coverage Status](https://coveralls.io/repos/github/neverendingqs/serverless-dotenv-plugin/badge.svg?branch=master)](https://coveralls.io/github/neverendingqs/serverless-dotenv-plugin?branch=master)
 [![npm version](https://img.shields.io/npm/v/serverless-dotenv-plugin.svg?style=flat)](https://www.npmjs.com/package/serverless-dotenv-plugin)
 
-Preload environment variables into serverless. Use this plugin if you have variables stored in a `.env` file that you want loaded into your serverless yaml config. This will allow you to reference them as `${env:VAR_NAME}` inside your config _and_ it will load them into your lambdas.
-
-**`serverless>=3.0.0` introduces changes that significantly impacts this plugin. I would love your feedback about this on the [discussion thread](https://github.com/neverendingqs/serverless-dotenv-plugin/discussions/155). See the discussion thread or the FAQ below for details on the impact of how env vars are loaded with `serverless>=2.26.0` and `serverless>=3.0.0`.**
+Preload environment variables into serverless. Use this plugin if you have variables stored in a `.env` file that you want loaded into your Lambda functions.
 
 ## Do you need this plugin?
 
-Changes in `serverless>=3.0.0` means this plugin can no longer preload environment variables into Serverless. You may want to consider an alternative, such as the one outlined in [`serverless-dotenv-example`](https://github.com/neverendingqs/serverless-dotenv-example).
+Serverless Framework can now natively resolve `${env:xxx}` variables from `.env` files by setting [`useDotenv: true` in the configuration](https://www.serverless.com/framework/docs/environment-variables):
+
+```yaml
+useDotenv: true
+
+provider:
+  environment:
+    FOO: ${env:FOO}
+```
+
+This plugin is still useful if you want to automatically import **all** variables from `.env` into functions:
```

Comment:
Not sure if https://github.com/serverless/serverless is looking to support more scenarios (e.g. different `dotenv` files for different stages, different file paths). Until then, I would like to leave users with an alternative.

```suggestion
For more complex situations, you will need to [wire up `dotenv` yourself](https://github.com/neverendingqs/serverless-dotenv-example).

This plugin is only useful if you want to automatically import **all** variables from `.env` into functions:
```